stellar/deploy.testnet.sh

```
@@ -2,19 +2,20 @@ bash build.sh
 
 my_address=($(stellar keys address potlock-testnet))
 native_token=CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC
-lists_contract_id=($(stellar contract deploy --wasm  target/wasm32-unknown-unknown/release/lists.wasm --source potlock-testnet --network testnet))
-registry_contract_id=($(stellar contract deploy --wasm  target/wasm32-unknown-unknown/release/project_registry.wasm --source potlock-testnet --network testnet))
+lists_contract_id=($(stellar contract deploy --wasm  target/wasm32-unknown-unknown/release/lists_opt.wasm --source potlock-testnet --network testnet))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)